stake_vote/src/lib.rs

```diff
@@ -36,7 +37,9 @@ pub async fn start_stakes_and_votes_loop(
     rpc_client: Arc<RpcClient>,
     grpc_url: String,
 ) -> anyhow::Result<tokio::task::JoinHandle<()>> {
```

Comment:
Use AnyhowJoinHandle

Reply:
OK, I'll do the change in the next PR because I've done a lot of change since and it's easier to do it in the current version of the code.